src/compartment.c

```diff
@@ -935,6 +759,13 @@ map_comp_entry_points(struct Compartment *new_comp)
     }
 }
 
+/* Resolve relocations with compartment relative addresses
+ *
+ * The way AARCH64 relocations work is that they use relative addresses for the
```

Comment:
Is this AArch64 specific?

Reply:
I *think* it might be ARM64 specific, and just inherited in AARCH64. I also *think* it's different than `x86`, but I'm really unsure. I don't think it's worth checking, but I could add that to the comment, if you think it's misleading.